### ISSUE TITLE
Is there an option to turn off result printouts at every step?

### ISSUE THREAD

**Issue by ocolluphid** [tflearn/tflearn#852]
I'm running with the following settings: 
```
model.fit(X_train, Y_train, n_epoch = max_epochs, shuffle=True, snapshot_step= 100,
show_metric=True, batch_size=48, run_id='cifar10_cnn')
```
But still getting printouts of accuracy and loss statistics at every training step in spite of `snapshot_epoch=None`. Hence I'm assuming that the snapshots only refer to saving the model after certain intervals and printing some overall statistics after 100 steps. Would it be possible to add the option of limiting the printouts? I.e. perhaps only display results in the console after 100 steps or after each epoch. 

Thanks!

**Comment by aymericdamien** [comment]
Can you try to add `snapshot_epoch=False`? so the model should only be displayed, and saved, every 100 steps.

**Comment by ocolluphid** [comment]
Hi,
Thanks for your response!
I tried adding `snapshot_epoch = False` but I'm still getting step-by-step results. Note the special line inserted after 100 steps because I included the `snapshot_step= 100` specification:
```
Training Step: 290  | total loss: 0.24614 | time: 26.564s
| Adam | epoch: 001 | loss: 0.24614 - acc: 0.9340 -- iter: 18560/55000
Training Step: 291  | total loss: 0.23892 | time: 26.634s
| Adam | epoch: 001 | loss: 0.23892 - acc: 0.9359 -- iter: 18624/55000
Training Step: 292  | total loss: 0.23660 | time: 26.709s
| Adam | epoch: 001 | loss: 0.23660 - acc: 0.9360 -- iter: 18688/55000
Training Step: 293  | total loss: 0.23231 | time: 26.784s
| Adam | epoch: 001 | loss: 0.23231 - acc: 0.9378 -- iter: 18752/55000
Training Step: 294  | total loss: 0.22257 | time: 26.858s
| Adam | epoch: 001 | loss: 0.22257 - acc: 0.9409 -- iter: 18816/55000
Training Step: 295  | total loss: 0.20436 | time: 26.931s
| Adam | epoch: 001 | loss: 0.20436 - acc: 0.9468 -- iter: 18880/55000
Training Step: 296  | total loss: 0.19662 | time: 27.002s
| Adam | epoch: 001 | loss: 0.19662 - acc: 0.9474 -- iter: 18944/55000
Training Step: 297  | total loss: 0.19394 | time: 27.076s
| Adam | epoch: 001 | loss: 0.19394 - acc: 0.9480 -- iter: 19008/55000
Training Step: 298  | total loss: 0.20307 | time: 27.150s
| Adam | epoch: 001 | loss: 0.20307 - acc: 0.9454 -- iter: 19072/55000
Training Step: 299  | total loss: 0.21119 | time: 27.224s
| Adam | epoch: 001 | loss: 0.21119 - acc: 0.9493 -- iter: 19136/55000
Training Step: 300  | total loss: 0.20661 | time: 29.985s
| Adam | epoch: 001 | loss: 0.20661 - acc: 0.9497 | val_loss: 0.15920 - val_acc: 0.9617 -- iter: 19200/55000
--
```
